import.js

```diff
@@ -0,0 +1,228 @@
+function ajaxImport(artifact, objTemplate) {
+    $.ajax({
+        method: "GET",
+        crossDomain: true,
+        dataType: "json",
+        url: `${userInfo.spiraUrl}services/v5_0/RestService.svc/${artifact}${userInfo.auth}`,
+        success: function (data) {
+            let valueArray = [];
+            for (let obj in data) {
+                valueArray.push(jsonToArray(data[obj], objTemplate));
+            }
+            toExcel(artifact, valueArray);
+        },
+        error: function () {
+
+        }
+    });
+}
+
+function getComponents(project){
+    $.ajax({
+        method: "GET",
+        crossDomain: true,
+        dataType: "json",
+        url: `${userInfo.spiraUrl}services/v5_0/RestService.svc/projects/${project}/`
+        + `components?active_only=true&include_deleted=false&`
+        +`username=${userInfo.username}&api-key=${userInfo.apikey}`,
+        success: function (data) {
+            populateComponents(data);
+            enableButtons();
+        },
+        error: function () {
+            console.log("error retrieving components");
+            populateComponents([]);
+            enableButtons();
+        }
+    });
+}
+
+function getReleases(project){
+    $.ajax({
+        method: "GET",
+        crossDomain: true,
+        dataType: "json",
+        url: `${userInfo.spiraUrl}services/v5_0/RestService.svc/projects/${project}/releases${userInfo.auth}`,
+        success: function (data) {
+            populateReleases(data);
+            getComponents(project);
+        },
+        error: function () {
+            console.log("error retrieving releases");
+            populateReleases([]);
+            getComponents(project);
+        }
+    });
+}
+
+function getUsers(project){
+    $.ajax({
+        method: "GET",
+        crossDomain: true,
+        dataType: "json",
+        url: `${userInfo.spiraUrl}services/v5_0/RestService.svc/projects/${project}/users${userInfo.auth}`,
+        success: function (data) {
+            populateUsers(data);
+            getReleases(project);
+        },
+        error: function () {
+            console.log("error retrieving users");
+            populateUsers([]);
+            getReleases(project);
+        }
+    });
+}
+
+function jsonToArray(oldObj, objTemplate) {
+    let valArray = [];
+    for (let prop in objTemplate) {
+        objTemplate[prop] = oldObj[prop];
+        valArray.push(objTemplate[prop]);
+    }
+    return valArray;
+}
+
+function toExcel(artifact, newValues) {
+    return Excel.run(function (context) {
+        let sheetName = toSheetName(artifact);
+        let valRange = columnRanges[artifact] + (newValues.length + 2);
+        let sheet = context.workbook.worksheets.getItem(sheetName);
+        let sheetValues = sheet.getRange(valRange);
+        sheetValues.values = newValues;
+        enableButtons();
+        return context.sync();
+    });
+}
+
+function loadCustomFields(artifact, project) {
```

Comment:
another example of big function that should have comments to explain each section